c_binding/libsg.h

```diff
@@ -79,6 +79,12 @@
 #define _SG_T10_SPC_MODE_SENSE_MAX_LEN              0xffff
 /* ^ SPC-5 rev 12, 6.15 MODE SENSE(10) command introduction */
 
+#define _T10_SPC_LOG_SENSE_MAX_LEN                  0xffff
+/* ^ SPC-5 rev07 6.8 LOG SENSE command */
+
+#define _T10_SPC_REQUEST_SENSE_MAX_LEN              0xffff
```

Comment:
For `REQUEST SENSE`, the `ALLOCATION LENGTH` is 1 byte, which max length is 0xff.
I double checked all other XXX_MAX_LEN, they all fit into SCSI SPEC for 0xffff.